packages/rest-generator/src/parser/request-body.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { createLogger } from '@sap-cloud-sdk/util';
+import { createRefs } from '../../test/test-util';
+import { parseRequestBody } from './request-body';
+
+describe('getRequestBody', () => {
```

Comment:
Question: Is it possible to define a schema also inline like for the parameters? I do not see a test for that if this is possible.

Reply:
Yes it is, this would be a follow up then, because I did not yet check what the openapi generator does. I am not testing this for the parameters either (I think).